### ISSUE TITLE
Make gives me errors

### ISSUE THREAD

**Issue by hcgilje** [pierrep/lpmt#7]
Hi, 
I am trying to install this on ubuntu in the terminal with make, but get this error:
```
Reading makefile 'Makefile'...
Reading makefile 'config.make' (search path) (no ~ expansion)...
Reading makefile '//libs/openFrameworksCompiled/project/makefileCommon/compile.p
roject.mk' (search path) (no ~ expansion)...
Updating makefiles....
 Considering target file '//libs/openFrameworksCompiled/project/makefileCommon/c
ompile.project.mk'.
  File '//libs/openFrameworksCompiled/project/makefileCommon/compile.project.mk'
 does not exist.
```

...and then a long list of errors because it doesn't find this file.

**Comment by pierrep** [comment]
I'll have a test, l haven't built it using the makefile in a while, however that looks more like a problem with your openFrameworks setup. Do you have OF installed? I'm planning on making an AppImage for Linux eventually....

**Comment by pierrep** [comment]
I had a test and the Makefile works fine, sounds like you don't have OF set up. The set up guides are on the download page - https://openframeworks.cc/download/

Let me know if you need any other assistance...

**Comment by hcgilje** [comment]
Hi, my bad.
I realized when I woke up this morning why this happened, and why I stopped using openframeworks :) Always some dependencies issues when trying to compile other people's projects, but I will give it a try.
Maybe put a little note in the readme that openframeworks needs to be installed? It's obvious for OF users, but not for people searching for a projection mapping software.
It would be fantastic if you got around to making an App Image. As far as I can see lpmt seems to be the best alternative on linux for projection mapping, but difficult to use in a workshop setting if it needs to be installed on various computers.
I really appreciate that you have revived this project! 
I made VPT for mac and windows (based on Max) many years ago, and people still use it, so there is definitely room for free video projection software. It would be great to have an easy to use linux alternative, so I will check in here now and then to see if you have made an appimage version.

**Comment by pierrep** [comment]
Yes I just updated the instructions to make it clearer.
You'll find no problems compiling this one, just get the https://github.com/danomatika/ofxMidi addon as per the instructions, All the other addons are "cached" in the source code.
I'm definitely keen to create an AppImage, I'm a full-time Linux user and I've done lots of workshops with LPMT so it would definitely be useful. I'm in the last stretch of my PhD though at the moment, ahaha, so I'll see how it goes.

**Comment by hcgilje** [comment]
Wow,
this was the smoothest OF install I have ever done (my previous linux attempt was on a raspberry pi). I managed to compile lpmt as well and get the app running. Looking forward to exploring it!

**Comment by pierrep** [comment]
Feel free to make any suggestions once you give it a spin!

**Comment by pierrep** [comment]
OK so I procrastinated and made an AppImage. Need to test on different distros...if you want to try it's here:

https://github.com/pierrep/lpmt/releases/tag/v1.0.0

**Comment by hcgilje** [comment]
Sorry to have distracted you from phd work, but great that you managed to make an Appimage!
I am currently living a schizophrenic life switching between macOS, windows and linux, so I am up for cross platform testing if needed.

**Comment by pierrep** [comment]
Oh by the way I realised I did have some binaries for OSX and Windows that I forgot I made. I've added them to the release page

**Comment by hcgilje** [comment]
I tried the mac binary, but it just closes immediately when I open it (I am on 10.14)

**Comment by pierrep** [comment]
It's a while since I built it, but it definitely worked for me then. re you able to run it from a terminal to get the exact error mesage?
